### PR TITLE
Add RTL language support on frontend, tidy up sass

### PR DESCRIPTION
**Pull request by eadpearce**
https://uktrade.atlassian.net/browse/CMS-19

screencapture-localhost-8011-ar-industries-energy-2018-05-02-15_05_21
screencapture-localhost-8011-ar-2018-05-02-15_06_31

### CODE REVIEW

File: invest/templates/includes/head.html [hunk 1]
```diff
@@ -24,14 +27,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <!-- Bootstrap core CSS -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
-          integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-
+    {% if LANGUAGE_BIDI %}
+      <link rel="stylesheet" href="https://cdn.rtlcss.com/bootstrap/v4.0.0/css/bootstrap.min.css" integrity="sha384-P4uhUIGk/q1gaD/NdgkBIl3a6QywJjlsFJFk7SPRdruoGddvRVSwv5qFnvZ73cpz" crossorigin="anonymous">
```
Comment (richtier) [inline invest/templates/includes/head.html]: rtlcss.com/ TIL :)

File: invest/templates/includes/head.html [hunk 2]
```diff
@@ -24,14 +27,19 @@
...
+    {% else %}
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    {% endif %}
     <!-- Fonts -->
-    <link href='https://fonts.googleapis.com/css?family=Overpass:|Roboto:400,700,300' rel='stylesheet' type='text/css'>
+    <link href="https://fonts.googleapis.com/css?family=Overpass:|Roboto:400,700,300" rel="stylesheet" type="text/css">
```
Comment (richtier) [inline invest/templates/includes/head.html]: we don't use Overpass on any other service. can you confirm with michal if we want to use it here

Comment (eadpearce) [reply invest/templates/includes/head.html]: ??? We use it on the hero headings on all services

Comment (eadpearce) [reply invest/templates/includes/head.html]: (also that font weight is wrong i'll fix it)

File: invest/templates/includes/head.html [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 {% load wagtailcore_tags wagtailimages_tags %}
+{% load i18n %}
+{% get_current_language_bidi as LANGUAGE_BIDI %}
+{% get_current_language as LANGUAGE %}
 <!DOCTYPE html>
 <!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js" dir="{{ LANGUAGE_BIDI|yesno:'rtl,ltr' }}" lang="{{ LANGUAGE }}"> <!--<![endif]-->
```
[the remark this answers is not in the document]
Comment (eadpearce) [reply invest/templates/includes/head.html]: I just copied what you did in FAS but sure I can get rid of the other tags

**Pull request merged**